src/SwiftUI/iOS/Color.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+using Foundation;
+using UIKit;
+
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	public unsafe partial class Color
```

Comment:
Please see the comments on the Mac version for this.. I think those all apply here too

Reply:
`SwiftUI.String` issue addressed and please see my comments about `NSBundle`